### PR TITLE
feat(theme): add an accordion component

### PR DESCRIPTION
**Pull request by lexzhukov**
### Please read and mark the following check list before creating a pull request:

 - [] I read and followed the [CONTRIBUTING.md](https://github.com/akveo/nebular/blob/master/CONTRIBUTING.md) guide.
 - [] I read and followed the [New Feature Checklist](https://github.com/akveo/nebular/blob/master/DEV_DOCS.md#new-feature-checklist) guide.
 
 #### Short description of what this resolves:

### CODE REVIEW

File: src/playground/playground-routing.module.ts [hunk 1]
```diff
@@ -337,6 +338,15 @@ export const routes: Routes = [
           },
         ],
       },
+      {
+        path: 'accordion',
+        children: [
+          {
+            path: '',
```
Comment (nnixaa) [inline src/playground/playground-routing.module.ts]: path should reflect the file name (by convention)

File: src/app/app.module.ts [hunk 2]
```diff
@@ -15,9 +16,12 @@ import { NbAppComponent } from './app.component';
 import { NbLayoutDirectionToggleComponent } from './layout-direction-toggle/layout-direction-toggle.component';
 import { NbDynamicToAddComponent } from '../playground/shared/dynamic.component';
 import { NbPlaygroundSharedModule } from '../playground/shared/shared.module';
+import { NbLayoutThemeToggleComponent } from './layout-theme-toggle/layout-theme-toggle.component';
+
 @NgModule({
   imports: [
     BrowserModule,
+    BrowserAnimationsModule,
```
Comment (nnixaa) [inline src/app/app.module.ts]: should we add it here or to playground.module?

File: src/framework/theme/components/accordion/_accordion-item-description.component.theme.scss [hunk 3]
```diff
@@ -0,0 +1,11 @@
+/**
+ * @license
+ * Copyright Akveo. All Rights Reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+@mixin nb-accordion-item-description-theme() {
+  nb-accordion-item-description {
+    color: nb-theme(accordion-item-description-color-fg);
```
Comment (nnixaa) [inline src/framework/theme/components/accordion/_accordion-item-description.component.theme.scss]: no need for `color`, just `accordion-item-description-fg`

File: src/app/app.module.ts (hunk 2 above)
Comment (lexzhukov) [reply src/app/app.module.ts]: `BrowserAnimationsModule` can be imported only in `AppModule`.
https://github.com/telerik/kendo-angular/issues/454#issuecomment-323723592

**Pull request closed without merging**